Mathlib/Order/Bounds/Basic.lean

```diff
@@ -974,7 +974,7 @@ theorem mem_lowerBounds_image (Has : a ∈ lowerBounds s) (Hat : a ∈ t) :
 theorem mem_lowerBounds_image_self : a ∈ lowerBounds t → a ∈ t → f a ∈ lowerBounds (f '' t) :=
   Hf.mem_lowerBounds_image subset_rfl
 
-theorem image_upperBounds_subset_upperBounds_image (Hst : s ⊆ t) :
+theorem image_upperBounds_subset_upperBounds_image (Hf : MonotoneOn f t) (Hst : s ⊆ t) :
```

Comment:
Perhaps a follow-up question: did you also mean to delete the `(Hf : MonotoneOn f t)` in the variables declaration 17 lines up?

Reply:
No, that needs to stay: most of the theorems here pick this up in term mode, I've just added to the places that only pick it up from tactic mode.

This may all need revisiting after #2452 is resolved.